tox.ini

```diff
@@ -30,4 +30,23 @@ passenv =
 commands =
     clean: python -c 'import shutil; [shutil.rmtree(p, True) for p in ("build", "dist", "docs/_build")]'
     clean: python -c 'import pathlib, shutil; [shutil.rmtree(p, True) for p in pathlib.Path("src").glob("*.egg-info")]'
-    build: python -m build {posargs}
\ No newline at end of file
+    build: python -m build {posargs}
+
+
+[testenv:publish]
+description =
+    Publish the package you have been developing to a package index server.
+    By default, it uses testpypi. If you really want to publish your package
+    to be publicly accessible in PyPI, use the `-- --repository pypi` option.
+skip_install = True
+changedir = {toxinidir}
+passenv =
+    # See: https://twine.readthedocs.io/en/latest/
+    TWINE_USERNAME
+    TWINE_PASSWORD
```

Comment:
Do you want me to add these envs as secrets to the repo? Or do you intend we use the Trusted Publisher from pypi? Either way, let me know what this workflow requires and I will set it up.

Reply:
Thanks for bringing this up! I suggest setting this repo to be a Trusted Publisher.

Reply:
Just configured this repo as a pending Trusted publisher. Pypi part should be good to go.